s4e-web/cypress/integration/user-options/user-options-mail-to-expert.spec.ts

```diff
@@ -15,10 +15,16 @@ describe('Mail to Expert', () => {
   });
 
   it('should send mail to expert', () => {
+    cy.deleteAllMails();
+
     UserOptionsMailToExpert.openSendMailToExpertModal()
       .addMessageToSupport('Wsparcie zdalne', 'Test Message')
       .sendMessageToSupport();
     ConfirmModal.accept();
     UserOptionsMailToExpert.confirmationShouldToAppear();
+
+    cy.getMailBySubject('Prośba');
```

Comment:
I'm not sure if it will work correctly due to invalid chars encoding in MailHog JSON.

Reply:
'Prosba' doesnt work